dev-docs/source/RunningTheUnitTests.rst

```diff
@@ -103,31 +103,33 @@ Starting in your build folder (e.g. Mantid/Code/debug):
 
    -  Not possible with ctest.
 
-Visual Studio/ XCode note
-#########################
+Running Unit Tests from Within Visual Studio
+############################################
 
-In Visual Studio the user can alter the properties of the subset of
-tests (inside the unitTest directory (e.g. AlgorithmTest). In the
-properties box it is possible to specify a specific test to run by
-typing its name in the TargetName box. Then to execute the test, right
-click the subset of tests and select debug and then start new instance.
+Open the Mantid solution in Visual Studio 2015. To run a subset of tests (for example ``UnitTests/AlgorithmsTest``);
 
-To run the tests under one of these environments then you will need to
-open a command window and change to the build directory. Once there you
-can run the tests by selecting the configuration;
+-  In the Solution Explorer, right click the top level directory for the tests (in this case ``UnitTests/AlgorithmsTest``) and select Properties.
 
-.. code-block:: sh
+-  In the Debugging tab of Properties change the Command Arguments box to the name of the test, for example "AddNoteTest".
 
-   ctest -C Debug -j4
+-  Right click the directory again and select Debug->Start new instance.
 
-This runs all tests in Debug mode (note that this will NOT build any
-outdated libraries). To select a subset use the ``-R`` option:
+   Once the build has finished, open a file browser and navigate to the mantid build directory, run the command-prompt.bat file to open a command prompt and run 
```

Comment:
Can we make a separation here between running entirely within Visual Studio, i.e. "Debug->Start new instance" and on the command line using `ctest`? `ctest` is not necessary for running inside Visual Studio and if you want to use the command line then clicking build on the project and then running `ctest` will work.